opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -3230,4 +3263,140 @@ namespace Opm
         }
     }
 
+
+
+
+
+
+    template<typename TypeTag>
+    void
+    MultisegmentWell<TypeTag>::
+    assembleSICDPressureEq(const int seg) const
```

Comment:
Method argument name should make it clear that it takes a segment index, not a segment number (if I understood this correctly).

Reply:
Please let me know whether it is acceptable to you. `segment_number` will always be referred to `***_number`.  `seg` has always been a index.  If you think it is not clear enough, I can replace all the `seg`s.

Reply:
As long as it is consistent, that is fine.